tests/testthat/test-augment.R

```diff
@@ -26,6 +26,11 @@ test_that("augment fit_resamples", {
   expect_true(sum(names(aug_1) == ".resid") == 0)
   expect_s3_class_bare_tibble(aug_1)
 
+  expect_identical(
+    rle(substr(names(aug_1), 1, 1) == ".")$values,
+    c(TRUE, FALSE)
+  )
+
```

Comment:
This is a (maybe too) fancy way of checking that columns that start with `.` appears first

Reply:
it's fine. I usually use `grepl("^\\.", x)`